CHANGELOG.md

```diff
@@ -15,6 +16,8 @@
 
 ### Deprecated
 
+- [PR #422](https://github.com/konpyutaika/nifikop/pull/422) - **[Operator/NifiCluster]** `MaximumEventDrivenThreadCount` is no longer configurable in NiFi.
```

Comment:
This should be removed from the nifi-cluster helm chart as well.

Reply:
Should be okay now.